### ISSUE TITLE
After ending session, unable to join conference with new session

### ISSUE THREAD

**Issue by marymhart** [voxeet/voxeet-uxkit-reactnative#12]
**Describe the bug**

After ending a session, I am unable to start a new session & join new conference.

**Expected Behaviour**
After ending a session, I should be able to open a new session & start a new conference. 

**Steps to Reproduce the Problem**

  1. Create conference using & end conference using `handleCreateConference()` & `handleEndConference()` below
  2. Start a new conference using `handleCreateConference()` & user is unable to join. Conference is successfully created but no error is returned from the join. The Dolby dashboard shows no join request & no users in the conference.

```
export const handleCreateConference = async ({
  user,
  conferenceAlias,
  consumerToken,
  refreshToken,
}) => {
  try {
    await handleInitializeVoxeet({
      consumerToken,
      refreshToken,
    });
    const conferenceUser =
      Platform.OS === 'android'
        ? {
            ...user,
            externalId: '',
          }
        : ConferenceUser('', `${user.name}`, `${user.avatarUrl || ''}`);
    await VoxeetSDK.connect(conferenceUser);
    const conference = await VoxeetSDK.create({
      alias: conferenceAlias,
      params: {
        videoCodec: 'H264',
      },
    });
    const conferenceOptions =
      Platform.OS === 'android'
        ? {
            user,
            alias: conferenceAlias,
          }
        : {
            user: {
              ...conferenceUser,
              type: 'user',
            },
            alias: conference.conferenceAlias,
          };
    return await VoxeetSDK.join(conference.conferenceId, conferenceOptions);
  } catch (err) {
    return err;
  }
};

export const handleEndConference = async () => {
  await VoxeetSDK.leave();
  return await VoxeetSDK.disconnect();
};
```

[Please provide a link to a repository showing specifically the issue. It should only have the minimal footprint of your own implementation - something which does not requires any NDA.]

**Specifications**

  - Device: iOS simulator & device
  - Platform: iOS
  - Version: 
    "@voxeet/react-native-voxeet-conferencekit": "0.1.29"
    "react": "16.9.0",
    "react-native": "0.61.5",

**Comment by vjard** [comment]
Hello @marymhart,

Does this code call VoxeetSDK.initialize("xxx", "xxx") ?
```
await handleInitializeVoxeet({
      consumerToken,
      refreshToken,
    });
```

If this is the case, you don't have to call this code every time you want to create a conference, but only once. 
I managed to reproduce your problem by calling this code several times. Can you try it by calling it only once during your application cycle?

**Comment by marymhart** [comment]
hey @vjard the function is below. The main issue we are facing is when we log out & log back in of our app. In this instance we do want to call `initializeToken` again since the token will not be refreshed while logged out. Is there a rate limit or something we should be aware of?

```
export const handleInitializeVoxeet = (data) => {
  if (Platform.OS === 'android') {
    return handleInitializeToken(data);
  } else {
    return VoxeetSDK.isUserLoggedIn()
      .then((userLoggedIn) =>
        !userLoggedIn ? handleInitializeToken(data) : userLoggedIn,
      )
      .catch((err) => err);
  }
};

export const handleInitializeToken = (data) =>
  VoxeetSDK.initializeToken(data.consumerToken, () =>
    handleRefreshToken(data.refreshToken, data.rhinovideoUrl),
  );
```

**Comment by vjard** [comment]
Hello @marymhart,

We have just fixed this issue in version 0.1.31 (now available)
Thank you for your feedback.
